charts/kong/values.yaml

```diff
@@ -22,6 +22,7 @@ deployment:
   ## Specify the service account to create and to be assigned to the deployment / daemonset and for th migrations
   serviceAccount:
     create: true
+    automountServiceAccountToken: false
```

Comment:
Would you mind adding a comment describing the property?

Reply:
Added, thank you for pointing that out!